### PR TITLE
Add optimizations from 1.10 to 1.11 and 1.12 IWorldHandler.

### PR DESCRIPTION
**Pull request by HumorousFool**
### Describe in detail what your pull request acomplishes
This PR adds the block overwrite check from the 1.10 IWorldHandler to the 1.11 and 1.12 version.

### Checklist
- [x] Proper internationalization
- [x] Compiled/tested

### CODE REVIEW

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/ManOverboardCommand.java [hunk 1]
```diff
@@ -57,6 +57,7 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
         }
 
         player.setVelocity(new Vector(0, 0, 0));
+        player.setFallDistance(0);
```
Comment (TylerS1066) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/ManOverboardCommand.java:60]: Let's keep changes in separate PRs

**Review by TylerS1066 — CHANGES_REQUESTED** [review]

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/ManOverboardCommand.java (hunk 1 above)
Comment (HumorousFool) [reply modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/ManOverboardCommand.java:60]: If by that you mean the manoverboard change, that's a mistake, I'm new to github, I'll remove this file from the PR.

**Comment by HumorousFool** [comment]
Requested changes made

**Review by TylerS1066 — APPROVED** [review]

**Comment by HumorousFool** [comment]
Does this also need ccm's approval?

**Comment by TylerS1066** [comment]
> Does this also need ccm's approval?

Due to the fact that the world handlers are almost solely his/yarro's code and I do not understand them, I would like @cccm5 's review.

**Review by cccm5 — APPROVED** [review]
